paasta_tools/deployd/metrics.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import time
+
+from paasta_tools.deployd.common import PaastaThread
+
+try:
+    import yelp_meteorite
+except ImportError:
+    # Sorry to any non-yelpers but you won't
+    # get metrics emitted as our metrics library
+    # is currently not open source
+    yelp_meteorite = None
+
+_metricis_interfaces = {}
+
+
+def register_metrics_interface(name):
+    def outer(func):
+        _metricis_interfaces[name] = func
+        return func
+    return outer
+
+
+def get_metrics_interface(name):
+    return _metricis_interfaces[name]()
+
+
+@register_metrics_interface(None)
+class NoMetrics(object):
+    def create_timer(self, *args, **kwargs):
```

Comment:
Yeah, I guess. I was trying to use kwargs at least because it makes it easy to add dimensions as kwargs when you call this from the code...